administration/src/bp-modules/NavigationBar.tsx

```diff
@@ -67,6 +67,11 @@ const Navigation = (props: Props) => {
             <Button minimal icon='path-search' text='Region verwalten' />
           </NavLink>
         ) : null}
+        {role === Role.ProjectStoreManager ? (
```

Comment:
:question: Shouldn't this be available to at least project admins as well? Applies to later usages as well.

:question: Is there documentation somewhere about the different roles and their permissions?

Reply:
its in the backend Role class
no at the moment this is only for our usage as explained in the description so no one should be able to create this role for now except via backend command line

Reply:
Thanks for explaining :)